docs/reference/query-dsl/geo-bounding-box-query.asciidoc

```diff
@@ -52,8 +52,9 @@ Then the following simple query can be executed with a
 |Option |Description
 |`_name` |Optional name field to identify the filter
 
-|`ignore_malformed` |Set to `true` to
-accept geo points with invalid latitude or longitude (default is `false`).
+|`validation_method` |Set to `IGNORE_MALFORMED` to
+accept geo points with invalid latitude or longitude, set to
+`COERCE` to also try to infer correct latitude or longitude. (default is `STRICT`).
```

Comment:
Should we add deprecation warnings in the asciidoc to indicate that ignore_malformed and coerce are replaced by validation_method?